sosw/components/test/unit/test_helpers.py

```diff
@@ -666,5 +666,23 @@ def test_to_bool(self):
             to_bool(object())
 
 
+    def test_get_message_dict_from_sns_event(self):
+        event_true = {'Records': [{'Sns': {'Message': '{}'}}]}
+        event_false = {'Records': [{'Sns': {'Message': ''}}]}
+        event_empty = {'Records': [{'Sns': {}}]}
+
+        self.assertEqual(get_message_dict_from_sns_event(event_true), {})
+        self.assertRaises(ValueError, validate_account_to_dashed, event_false)
```

Comment:
`validate_account_to_dashed` has nothing to do here. Please update the tests appropriately and have an easy way to add more edge-cases please. TESTS_TRUE = {...}, TESTS_FALSE = {...} etc